syslogish/server.go

```diff
@@ -16,7 +16,7 @@ const (
 	bindHost          = "0.0.0.0"
 	bindPort          = 1514
 	queueSize         = 500
-	controllerPattern = `^(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
+	controllerPattern = `(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
```

Comment:
Im not sure wtf happen here. I tested the original regex and everything worked just fine then I changed it from a string to this and it broke. Here is debuggex to show you sample output and the regex to test against.. https://www.debuggex.com/r/VZbX7F5EqhSJ3Dhh